packages/svelte/src/lib/types.ts

```diff
@@ -1,10 +1,10 @@
 import type { PageProps } from '@inertiajs/core'
 import type { ComponentType } from 'svelte'
 
-export type ComponentResolver = (name: string) => ComponentType | Promise<ComponentType>
+export type ComponentsResolver = (name: string) => ResolvedComponents | Promise<ResolvedComponents>
```

Comment:
This should remain singular if I'm understanding correctly, yes? `ComponentResolver`, since it resolves a single component?

Reply:
That depends on your answer below about the `ResolvedComponents` :)

Reply:
It's done! :)